src/main/java/pianokeys/IconButton.java

```diff
@@ -0,0 +1,75 @@
+package pianokeys;
+
+import javax.swing.*;
+import java.awt.*;
+
+import static java.awt.Color.WHITE;
+
+public class IconButton extends JButton
```

Comment:
I admire you trying to draw the icons, but just find icons online and download them